templates/post_batched_post_generic.html

```diff
@@ -0,0 +1,37 @@
+{{define "post_batched_post_generic"}}
```

Comment:
This is the same as `post_batched_post_full`, except that it doesn't include the notification contents